prov/efa/test/efa_unit_test_ep.c

```diff
@@ -246,48 +250,6 @@ void test_efa_rdm_ep_handshake_receive_without_peer_host_id_and_do_not_send_loca
 	test_efa_rdm_ep_handshake_exchange_host_id(state, 0x0, 0x0, true);
 }
 
-/**
- * @brief Test efa_rdm_ep_open() handles rdma-core CQ creation failure gracefully
- *
- * @param[in]	state		struct efa_resource that is managed by the framework
- */
-void test_efa_rdm_ep_cq_create_error_handling(struct efa_resource **state)
```

Comment:
moved this test to efa_unit_test_cq.c as we are creating ibv cq in fi_cq_open now.